data/transactions/logic/TEAL_opcodes.md

```diff
@@ -1032,7 +1032,7 @@ params: Txn.Accounts offset (or, since v4, an _available_ account address), _ava
 
 ## pushbytes bytes
 
-- Opcode: 0x80 {varuint length} {bytes}
+- Opcode: 0x80 {varuint count} {bytes}
```

Comment:
I don't think this one should be changed.  It's the length of the byte array, not the count of how many byte arrays there are.